GitUI/BranchTreePanel/RepoObjectsTree.Nodes.Submodules.cs

```diff
@@ -91,9 +96,33 @@ protected override string NodeName()
 
             public void Open()
             {
+                if (Info?.Detailed?.RawStatus != null)
+                {
+                    UICommands.BrowseSetWorkingDir(Info.Path, ObjectId.WorkTreeId, Info.Detailed.RawStatus.OldCommit);
+                    return;
+                }
+
                 UICommands.BrowseSetWorkingDir(Info.Path);
             }
 
+            public void OpenGE()
+            {
+                var first = string.IsNullOrWhiteSpace(Info?.Detailed?.RawStatus?.OldCommit?.ToString())
+                    ? string.Empty
+                    : $",{Info?.Detailed?.RawStatus.OldCommit}";
+
+                var process = new Process
+                {
+                    StartInfo =
+                    {
+                        FileName = Application.ExecutablePath,
+                        Arguments = $"browse -commit={ObjectId.WorkTreeId}{first}",
```

Comment:
💭 I think we ought to centralise GE cli commands, we have 3 or 4 instances of this line throughout the codebase.

Reply:
I had prepared this: #8738

Reply:
https://github.com/gitextensions/gitextensions/pull/8623/commits/be250ea104ecb396ca49a326b82da4785675d724

Reply:
Will be handled outside this PR